### PR TITLE
Homework #2

### PR DESCRIPTION
**Pull request by ghost**
Kozoriezov V.

### CODE REVIEW

File: lib/topics/state_and_behaviour.rb [hunk 1]
```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class Car
+  class Car
+    attr_reader :color, :year, :model, :current_speed
+
+    def initialize(car_attributes)
+      @color = car_attributes.fetch(:color, 'black')
```
Comment (sveredyuk) [inline lib/topics/state_and_behaviour.rb]: 'black' and all default values to CONSTANTS

File: lib/topics/state_and_behaviour.rb [hunk 2]
```diff
@@ -2,4 +2,27 @@
...
+      @model = car_attributes.fetch(:model, 'tesla_model_3')
+      @current_speed = nil
+    end
+
+    def self.default_car
+      Car.new
+    end
+
+    def speed_up(speed)
+      @current_speed = speed
```
Comment (sveredyuk) [inline lib/topics/state_and_behaviour.rb]: maybe increase speed? `@current_speed += speed` (and try `more_speed` as argument name. more human readable)

File: lib/topics/state_and_behaviour.rb [hunk 3]
```diff
@@ -2,4 +2,27 @@
...
+    end
+
+    def push_break(negative_velocity)
```
Comment (sveredyuk) [inline lib/topics/state_and_behaviour.rb]: negative_velocity -> less_speed

File: lib/topics/state_and_behaviour.rb (hunk 2 above)
Comment (harusan) [inline lib/topics/state_and_behaviour.rb]: if I try increase - I have error
` 1) Error:
Car::#speed_up#test_0002_returns full info:
NoMethodError: undefined method `+' for nil:NilClass
    /home/harusan/RubymineProjects/homework2/lib/topics/state_and_behaviour.rb:25:in `speed_up'
    /home/harusan/RubymineProjects/homework2/test/topics/test_state_and_behaviour.rb:79:in `block (3 levels) in <class:StateAndBehaviourTest>'`

**Review by oleksandr-kychun — APPROVED** [review]

**Pull request closed without merging**